src/eip4844/eip4844.c

```diff
@@ -417,26 +410,22 @@ C_KZG_RET compute_kzg_proof(
  *
  * @param[out]  proof_out   The combined proof as a single G1 element
  * @param[out]  y_out       The evaluation of the polynomial at the evaluation point z
- * @param[in]   polynomial  The polynomial in Lagrange form
+ * @param[in]   poly        The polynomial in Lagrange form
  * @param[in]   z           The evaluation point
  * @param[in]   s           The trusted setup
  */
 static C_KZG_RET compute_kzg_proof_impl(
-    KZGProof *proof_out,
-    fr_t *y_out,
-    const Polynomial *polynomial,
-    const fr_t *z,
-    const KZGSettings *s
+    KZGProof *proof_out, fr_t *y_out, const fr_t *poly, const fr_t *z, const KZGSettings *s
 ) {
     C_KZG_RET ret;
     fr_t *inverses_in = NULL;
     fr_t *inverses = NULL;
+    fr_t *q = NULL;
```

Comment:
We could consider renaming this to `q_poly` or `quotient_poly`, to match the convention of renaming all polynomials to `poly` instead of using single-letter variable names.

Reply:
I agree. Good suggestion. Will make this change in a bit.

Reply:
Doing it and pushing here.